### PR TITLE
 Updated README to current state

### PR DESCRIPTION
**Pull request by aashish24**

### CODE REVIEW

**Review by manthey — APPROVED** [review]

**Comment by aashish24** [comment]
@manthey some more minor changes,  let me know if the latest changes makes sense

File: README.md [hunk 1]
```diff
@@ -29,6 +26,7 @@ Additional features includes [Annotations](https://opengeoscience.github.io/geoj
 [Animated Transitions](https://opengeoscience.github.io/geojs/examples/transitions), and
 [Widgets](https://opengeoscience.github.io/geojs/examples/widgets).
 
+GeoJS can also be used for visualizing medical imagery. See [HistomicsTK](https://github.com/DigitalSlideArchive/HistomicsTK) for an example.
```
Comment (manthey) [inline README.md:29]: We should probably point this to [Digital Slide Archive](https://digitalslidearchive.github.io/digital_slide_archive)

File: README.md [hunk 2]
```diff
@@ -52,7 +50,11 @@ if you have general questions or bug reports on GeoJS.
 
 Get Involved
 ============
-GeoJS is developed to meet needs of developers and domain-specific researchers,
-and we are always open for collaboration. We have been doing open source development
-for the last 17 year and understand and appreciate what it takes to develop open
-source software. So fork GeoJS, do great things and let us know if we can help.
+We are always open for collaboration. Kitware has been doing open source development
+for the last 20 year and understand and appreciate what it takes to develop open
```
Comment (manthey) [inline README.md:54]: I think this reads better if `understand` and `appreciate` are `understands` and `appreciates`.

**Review by manthey — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (aashish24) [reply README.md]: makes sense, I changed it accordingly. Have a look and let me know if anything else we want to address.

**Review by manthey — APPROVED** [review]

**Pull request merged**